### PR TITLE
feat: split screen support

### PR DESCRIPTION
**Pull request by arashsheyda**
Split Screen Mode (ability to add a second page) resolves #182

https://github.com/nuxt/devtools/assets/38922203/0a769e3f-0a4a-4040-b50a-1c63acba0837

### CODE REVIEW

**Comment by antfu** [comment]
Nice idea! I pushed some UI changes to the branch.

A few blockers we need to clear before merging this

- Find a way to isolate the route state. Currently many tabs relies on route hash to preserve navigation state. In the splitscreen will also update the hash, which will conflicts with the main view
- Improve some RWD for tabs on narrow width
- Able to make them work for custom iframe view as well (related to route path as the first point)

**Comment by arashsheyda** [comment]
@antfu thanks.
- aw yes. should we use useState or something like that?
- should I open a new PR for this?
- I think we can solve the `custom iframe` with passing `props`, should I push this?

https://github.com/nuxt/devtools/assets/38922203/b1b9b573-d330-47f4-be89-f738b7611471

**Comment by antfu** [comment]
Yeah I think each tab needs to be able to work in hash mode and memory (useState sounds good), controlled by the parent view.

Please update directly in this PR.

And for iframe, yes that sounds good

**Comment by arashsheyda** [comment]
I did refactore the hash & query to composable(state-rotues), I hope I didn't miss any
cc @antfu

File: packages/devtools/client/composables/state-routes.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// server-routes Tab
+export function useServerRoutesState() {
+  return useState<string>('server-routes:state')
```
Comment (antfu) [inline packages/devtools/client/composables/state-routes.ts:3]: Maybe we should use `useSessionStorage` to preserve the state across refreshes?

Comment (arashsheyda) [reply packages/devtools/client/composables/state-routes.ts]: Yes. that make sense. thank you

**Pull request merged**